src/cascadia/TerminalSettingsEditor/ColorToBrushConverter.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "ColorToBrushConverter.h"
+#include "ColorToBrushConverter.g.cpp"
+
+using namespace winrt::Windows;
+using namespace winrt::Windows::UI::Xaml;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    Foundation::IInspectable ColorToBrushConverter::Convert(Foundation::IInspectable const& value,
+                                                            Windows::UI::Xaml::Interop::TypeName const& /* targetType */,
+                                                            Foundation::IInspectable const& /* parameter */,
+                                                            hstring const& /* language */)
+    {
+        return winrt::box_value(Windows::UI::Xaml::Media::SolidColorBrush(winrt::unbox_value<Windows::UI::Color>(value)));
+    }
+
+    Foundation::IInspectable ColorToBrushConverter::ConvertBack(Foundation::IInspectable const& /*value*/,
+                                                                Windows::UI::Xaml::Interop::TypeName const& /* targetType */,
+                                                                Foundation::IInspectable const& /*parameter*/,
+                                                                hstring const& /* language */)
+    {
+        throw hresult_not_implemented();

```

Comment:
I thought you used two way bindings. Any chance this would cause a crash not being implemented the other way?

Reply:
So in ColorSchemes, the data I'm TwoWay binding on doesn't use any of the converters, and whoever does use the converters are only OneWay. It'll throw if you declare a TwoWay binding with this converter for sure. I'll definitely implement the ConvertBack when I need it for a TwoWay binding though.

Reply:
alrighty. thanks for clarity.